bundle_result.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package bundle_result;
+
+// Indicates the bundle successfully won its auction and was dispatched to the current leader.
+message Dispatched {
+  // The bundle's placement within the auction upon completion (e.g. 1st or 2nd place).
+  uint32 auction_placement = 1;
+
+  // This bundle's bid.
+  uint64 originator_bid_lamports = 2;
+
+  // Window of slots this bundle was sent for.
+  repeated uint64 slot_window = 4;
+
+  // List of validators this bundle was dispatched to.
+  repeated string validator_identities = 5;
+}
+
+// Indicates the bundle was dropped and therefore not forwarded to any validator.
+message Dropped {
+  oneof reason {
+    FailedSimulation failed_simulation = 1;
+    InternalError internal_error = 2;
+    LostAuction lost_auction = 3;
+  }
+}
+
+
+message LostAuction {
+  // Auctions unique identifier.
+  string auction_id = 1;
+
+  // Number of auction winners.
+  uint32 num_winners = 2;
+
+  // Total bundles competing in auction.
+  uint32 num_bundles = 3;
```

Comment:
is this the same as num_winners? when would num_winners != num_bundles?

Reply:
this is total bundles in the auction, num winners is total winners

Reply:
what's the difference? is num_winners the number of searchers?